docs/ship-postgres/README.md

```diff
@@ -0,0 +1,64 @@
+# Import Dashboards (Postgres Dump Method)
```

Comment:
Can we rename this topic as "Migrating RVF Metadata between instances".
file `docs/migrating-metadata.rst` . RST and md has almost same syntax

Discard section about import Dashboard
Add a section about Benefits towards end of the file.
- list the fix of dashboard import/export there

Also please add when we need to perform steps on target machine. before we install rvf or after. do we need to drop existing db etc.

Reply:
Sure!